bedrock/firefox/views.py

```diff
@@ -506,4 +506,19 @@ def new(request):
     if request.GET.get('product', None) or request.GET.get('os', None):
         return HttpResponsePermanentRedirect(reverse('firefox.new'))
 
-    return l10n_utils.render(request, 'firefox/new.html')
+    template = 'firefox/new.html'
+    scene = request.GET.get('scene', None)
+    version = request.GET.get('v', None)
+    dlstarted = request.GET.get('dlstarted', None)
+
+    # separate scenes version of /new
+    if version == '2':
+        if scene == '2':
+            template = 'firefox/new/scene2.html'
+        # REVIEW: Should this explicitly check for a None value instead
```

Comment:
I don't have a strong preference here, although I guess I would perhaps favor always showing the download button (scene1) in the event of an incorrectly typed link?